<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinScreen</class>
 <widget class="QWidget" name="WinScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="congratsLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Congratulations!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back to Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="smallGearImage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </widget>
  <widget class="QLabel" name="bigGearImage">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Big</string>
   </property>
  </widget>
  <widget class="QLabel" name="bigGearImage2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Big</string>
   </property>
  </widget>
  <widget class="QLabel" name="smallGearImage2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </widget>
  <widget class="QLabel" name="bigGearImage4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Big</string>
   </property>
  </widget>
  <widget class="QLabel" name="smallGearImage3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </widget>
  <widget class="QLabel" name="smallGearImage4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </widget>
  <widget class="QLabel" name="bigGearImage3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Big</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
